src/test/integration/L2VaultV2.int.sol

```diff
@@ -1,15 +1,17 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity =0.8.16;
 
-import {CommonVaultTest, ERC20} from "src/test/CommonVault.t.sol";
+import {CommonVaultTest, ERC20, TestPlus} from "src/test/CommonVault.t.sol";
 import {L2Vault} from "src/vaults/cross-chain-vault/L2Vault.sol";
 import {L2VaultV2} from "src/vaults/cross-chain-vault/L2VaultV2.sol";
 import {Vault} from "src/vaults/Vault.sol";
 import {VaultV2} from "src/vaults/VaultV2.sol";
 
+import {console2} from "forge-std/console2.sol";
+
 contract L2VaultV2_IntegrationTest is CommonVaultTest {
     function setUp() public virtual override {
-        vm.createSelectFork("polygon", POLYGON_FORK_BLOCK);
+        vm.createSelectFork("polygon", 50_951_000);
```

Comment:
why are we hardcoding the block number here instead of using `POLYGON_FORK_BLOCK`?

Reply:
We can revert back to Fixed Block, Changed it for Checking integration test with current state of the vault.